core/src/main/scala/cromwell/core/logging/JobLogger.scala

```diff
@@ -0,0 +1,26 @@
+package cromwell.core.logging
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingAdapter
+import cromwell.core.WorkflowId
+import org.slf4j.Logger
+
+trait JobLogging extends ActorLogging { this: Actor =>
+  def workflowId: WorkflowId
+  def jobTag: String
+
+  lazy val jobLogger: Logger = new JobLogger(self.path.name, workflowId, jobTag, Option(log))
+}
+
+/**
+  * Similar to WorkflowLogger, with the addition that the log tag includes the call tag.
+  */
+class JobLogger(name: String,
```

Comment:
FYI I actually stumbled upon 2 use cases (in the tests, and in Run.scala) where we need to use the constructor.